lib/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Util = require('util');
+
+const Config = require('../config');
+const MoviesServer = require('./server');
```

Comment:
Sorry, I missed this the first time, we usually align the equal signs for `require`s:

```js
const Config       = require('../config');
const MoviesServer = require('./server');
```